docs/spec/operations.md

```diff
@@ -23,8 +23,8 @@ Sidetree Methods MAY rely on the DID Document data model when verifying protocol
 
 Use the following process to generate a Sidetree-based DID:
 
-1. Generate a key pair using the defined [`KEY_ALGORITHM`](#key-algorithm), let this be known as the [update key pair](#update-key-pair).
```

Comment:
Do we want instead the notion of an operation key pair then an update key pair being a type of operation key pair?

Reply:
yes, IMO... the way the spec exists today, its ambigious... I unified to the defined term "operation key"... but imo it might be better to use the term "update key everwhere"... I would prefer to do that rename in a subsequent PR.